Utilities/sandbox/parse-manifest.sb

```diff
@@ -0,0 +1,19 @@
+(version 1)
+
+; Deny everything by default.
+(deny default)
+
+; Allow reading all files.
+(allow file-read*)
+
+; These are required by the Swift compiler.
+(allow process*)
+(allow sysctl*)
+(allow mach*)
+(allow signal)
+
+; Allow writing in temporary locations.
+(allow file-write*
+    (subpath "/private/var")
+    (subpath "/private/tmp")
+)
```

Comment:
I think we don't really need `/tmp`, I just added it to be safer (irony). Since the file is opened by SwiftPM and fd is passed to the subprocess, we don't really open a file for writing in `PackageDescription`. Overriding `TMPDIR` did not seem to make the difference. I did check if fopen fails inside PackageDescription.

 `/var` was needed by the compiler, I don't know what subdirectory does it actually need but saw the compiler cache moving (inside var subdirectories) when I was experimenting. IIRC it moved from /var/folders to /var/tmp at some point.